docs/sidebars.js

```diff
@@ -1,109 +1,111 @@
-/**
- The sidebars can be generated from the filesystem, or explicitly defined here.
- Create as many sidebars as you want.
- */
-
-// @ts-check
-
 /** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
 const sidebars = {
-  // By default, Docusaurus generates a sidebar from the docs folder structure
-  // tutorialSidebar: [{type: 'autogenerated', dirName: '.'}],
-
-  // But you can create a sidebar manually
   sidebarJSBolt: [
     {
-      type: 'doc',
-      id: 'index', // document ID
-      label: 'Bolt for JavaScript', // sidebar label
-      className: 'sidebar-title',
+      type: "doc",
+      id: "index",
+      label: "Bolt for JavaScript",
+      className: "sidebar-title",
+    },
+    "getting-started",
+    {
+      type: "category",
+      label: "Slack API calls",
+      items: ["concepts/message-sending", "concepts/web-api"],
+    },
+    {
+      type: "category",
+      label: "Events",
+      items: ["concepts/message-listening", "concepts/event-listening"],
+    },
+    {
+      type: "category",
+      label: "Interactivity & Shortcuts",
+      items: [
+        "concepts/acknowledge",
+        "concepts/shortcuts",
+        "concepts/commands",
+        "concepts/action-listening",
+        "concepts/action-respond",
+        "concepts/creating-modals",
```

Comment:
Would it be helpful to introduce another indentation level for modals? They are generally a more complex feature, and we have four docs related to them. WDYT?

Reply:
I try to avoid double nested navs because it turns into a lot of "i can't see what i want to find immediately", especially as people get used to having things not just on a single page. I think this is okay for now, but maybe worth thinking about pulling modals out to a new section